apps/agreement/contracts/Agreement.sol

```diff
@@ -1178,7 +1178,7 @@ contract Agreement is ILockManager, IAgreement, IArbitrable, IACLOracle, AragonA
     * @param _collateralToken Address of the ERC20 token to be used for collateral
     * @param _actionAmount Amount of collateral tokens that will be locked every time an action is submitted
     * @param _challengeAmount Amount of collateral tokens that will be locked every time an action is challenged
-    * @param _challengeDuration Challenge duration in seconds, during which the submitter can raise a dispute
+    * @param _challengeDuration Challenge duration, during which the submitter can raise a dispute
```

Comment:
Thought it was pointless to describe it in "seconds"; we should assume all Ethereum durations are seconds :)